src/spid-localstorage.js

```diff
@@ -25,7 +25,7 @@ function set(key, value, expiresInSeconds) {
 
 function clear(key) {
```

Comment:
Maybe, I wasn´t bold enough to make any changes more than needed.
I think the name might be there because it was called 'clear' in the cookie module.

If changed it think it should be changed there as well